salalib/pointdata.cpp

```diff
@@ -131,18 +131,18 @@ int PointMaps::addNewMap(const std::string& name)
    bool duplicate = true;
    while (duplicate) {
       duplicate = false;
-      for (size_t i = 0; i < size(); i++) {
-         if (at(i).getName() == myname) {
+      for (size_t i = 0; i < maps_vector.size(); i++) {

```

Comment:
for loops on standard containers can be written more concise and safer as
`for (auto& current : maps_vector)`